content/source/docs/cloud/workspaces/creating.html.md

```diff
@@ -5,6 +5,8 @@ page_title: "Creating Workspaces - Workspaces - Terraform Cloud and Terraform En
 
 # Creating Workspaces
 
+> For a hands-on introduction, try the [Get Started - Terraform Cloud guide](https://learn.hashicorp.com/terraform/cloud-getting-started/signup) on HashiCorp Learn.
```

Comment:
The other examples have "guide" outside the link.

```suggestion
> For a hands-on introduction, try the [Get Started - Terraform Cloud](https://learn.hashicorp.com/terraform/cloud-getting-started/signup) guide on HashiCorp Learn.
```

Reply:
Ah, good catch.